large_cohort_runner.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+"""
+Entry point to run the large cohort workflow.
+"""
+
+import os
+
+import click
+import coloredlogs
+
+from cpg_utils import to_path
+from cpg_utils.config import set_config_paths
+from cpg_workflows import defaults_config_path
+from cpg_workflows.stages.large_cohort import AncestryPlots, Frequencies, LoadVqsr
+from cpg_workflows.workflow import StageDecorator, run_workflow
+
+WORKFLOWS: dict[str, list[StageDecorator]] = {
+    "large_cohort": [
+        LoadVqsr,
+        Frequencies,
+        AncestryPlots,
+    ],
+}
```

Comment:
I'd be tempted to go nuclear here... the Workflows dict and a workflow CLI parameter are just for multi-pipeline flexibility. This whole script can be rewritten as:

```
    # even this section isn't necessary if you give the analysis runner argument `--config configs/defaults/large_cohort.toml `
    workflows = 'large_cohort'
    wfl_conf_path = to_path(__file__).parent / f"configs/defaults/{workflow}.toml"
    config_paths = os.environ["CPG_CONFIG_PATH"].split(",") + list(config_paths)
    set_config_paths(config_paths[:1] + [str(wfl_conf_path)] + config_paths[1:])

    # just run these stages
    run_workflow(stages=[LoadVqsr, Frequencies, AncestryPlots], dry_run=dry_run)
```

Reply:
Will get to this when large cohort is split out as it's own repo